public/index.php

```diff
@@ -1,10 +1,15 @@
 <?php
 
 use Core\Session;
+use Dotenv\Dotenv;
 
 const BASE_PATH = __DIR__ . '/../';
 require BASE_PATH . "vendor/autoload.php";
 
+// Carregar variáveis do .env
+$dotenv = Dotenv::createImmutable(__DIR__ . '/../');
+$dotenv->load();
+
```

Comment:
O ideal é sempre carregar na raiz do projeto, ou seja, no início para que todas as dependências tem acesso as envs.